tests/tracing/helper.py

```diff
@@ -113,9 +114,9 @@ def create_test_trace_info(
     )
 
 
-def get_traces() -> List[Trace]:
-    # Get all traces from the trace buffer
-    return list(TRACE_BUFFER.values())
+def get_traces(experiment_id=DEFAULT_EXPERIMENT_ID) -> List[Trace]:
```

Comment:
note: Changing this widely-used fixture to check backend traces instead of in-memory one, because the former is more important.